server/mongoDB.ts

```diff
@@ -24,7 +24,7 @@ if (MONGODB_DB === '') {
 type MongoCtx = {
   client: MongoClient;
   db: Db;
-  applicantDataCollection: Collection<User>;
+  dataCollection: Collection<User | SingletonInfo>;
```

Comment:
after adding back the old applicantDataCollection, add a new field, something like `singletonCollection: Collection<SingletonInfo>;`